### PR TITLE
Feature/avatar edit

### PR DESCRIPTION
**Pull request by lpyvnytska**

### CODE REVIEW

File: client/src/users/components/EditAvatarForm.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  font-weight: 400;
+  color: white;
+  background-color: rgba(22, 160, 133, 0.9);
+  display: inline-block;
+  padding: 4px 30px;
+  border-radius: 15px;
+  transition: all 0.2s;
+}
+
+.inputfile:focus + label,
```
Comment (yushevchuk) [inline client/src/users/components/EditAvatarForm.css]: Remove space, please.

File: client/src/users/components/EditAvatarForm.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            ref={fileInputRef}
+            className='inputfile'
+            accept='image/*'
+          />
+          <label htmlFor='file'>Chose New Image</label>
+        </div>
+        <div>
+          <label htmlFor='zoom'>Zoom:</label>
+
+          <input
```
Comment (yushevchuk) [inline client/src/users/components/EditAvatarForm.js]: Remove space, please.

File: client/src/users/components/EditAvatarForm.css (hunk 1 above)
Comment (lpyvnytska) [reply client/src/users/components/EditAvatarForm.css]: Ok. Removed

File: client/src/users/components/EditAvatarForm.js (hunk 2 above)
Comment (lpyvnytska) [reply client/src/users/components/EditAvatarForm.js]: Removed

**Pull request merged**